pkg/api/testing/fuzzer.go

```diff
@@ -201,6 +201,14 @@ func FuzzerFor(t *testing.T, version string, src rand.Source) *fuzz.Fuzzer {
 			c.FuzzNoCustom(s) // fuzz self without calling this function again
 			s.Type = api.SecretTypeOpaque
 		},
+		func(pv *api.PersistentVolume, c fuzz.Continue) {
+			c.FuzzNoCustom(pv) // fuzz self without calling this function again
+			pv.Status.Phase = api.VolumePending
```

Comment:
These are not fuzzing.  You should pick a random number [0-num_phases) and index into a table of phase strings.